Ports/tree/patches/ReadMe.md

```diff
@@ -0,0 +1,7 @@
+# Patches for tree on SerenityOS
+
+## `0001-Don-t-make-host-GCC-the-default.patch`
+
+Don't make host GCC the default
```

Comment:
Does it matter though, since we are setting `CC` anyways?

Reply:
yes, otherwise it doesnt let me use the serenity GCC binary, probably because they don't use `?=` sign so it can be overriden...

Reply:
I will put a better commit message on that patch though.